models/OrderDetail.js

```diff
@@ -0,0 +1,25 @@
+/* jshint indent: 1 */
+
+module.exports = function(sequelize, DataTypes) {
+	return sequelize.define('OrderDetail', {
+		order_id: {
+			type: DataTypes.INTEGER(11),
+			allowNull: false,
+			primaryKey: true,
+			references: {
+				model: 'OrderHeader',
+				key: 'id'
+			}
+		},
+		product_id: {
+			type: DataTypes.INTEGER(11),
+			allowNull: true
+		},
+		quantity: {
```

Comment:
There should be at least a default of zero

Reply:
Made it NOT NULL. Since its order quantity, 0 doesnt make sense either